app/views/layouts/provider/login.html.slim

```diff
@@ -1,12 +1,11 @@
 doctype html
-html lang="en"
+html lang="en" class="pf-m-redhat-font"
```

Comment:
Just to keep consistency, can we use the same format like the other 2 layouts?
`html[lang="en" class="pf-m-redhat-font"]`

Reply:
good point, updated!